src/main/kotlin/io/github/mojira/arisa/ModuleExecutor.kt

```diff
@@ -296,7 +296,7 @@ class ModuleExecutor(
 
         issues
             .filter { it.project.key in projects }
-            .filter { it.resolution?.name ?: "Unresolved" in resolutions }
+            .filter { it.resolution?.name ?: "unresolved" in resolutions }
```

Comment:
dont you need a .toLowerCase() in the name too?

Reply:
Ahh, I'm doing too many things at once xD
Thank you, though 😉